ColabTool/Shared/Post/story.jsx

```diff
@@ -13,7 +13,7 @@ const draftDetails = {
   createdAt: 'on March 2nd at 12:45pm (GMT)',
   email: 'ash@buffer.com',
   isRetweet: false,
-  postAction: 'This post is scheduled for 9:42pm (GMT)',
+  postAction: 'This post will be added to the queue',
```

Comment:
😂  Yes! I definitely thought that while going through and changing all these in the same way! Sounds perfect for a follow up 😄